dcos/tools.go

```diff
@@ -94,6 +115,9 @@ func (st *Tools) GetTimestamp() time.Time {
 
 // GetMasterNodes finds DC/OS masters.
 func (st *Tools) GetMasterNodes() (nodesResponse []Node, err error) {
+
+	start := time.Now()
```

Comment:
Hmm, might this be a prime example for using `defer`? 😄 

```go
	start := time.Now()
	defer func() {
		duration := time.Since(start)
		findMasterNodesHistogram.Observe(duration.Seconds())
	}()
```